alex/components/dm/base.py

```diff
@@ -214,14 +208,3 @@ def log_state(self):
         :return: none
         """
         self.dialogue_state.log_state()
-
-    def get_token(self):
```

Comment:
I didn't find it was being used anywhere, and I think I introduced it some time ago, so I thought I'd clean it up.